### ISSUE TITLE
WordCloud - Cannot read property 'label' of undefined

### ISSUE THREAD

**Issue by pravdinalex** [amcharts/amcharts4#1395]
Using `"@amcharts/amcharts4": "^4.5.3",`

creating WordCloud:

```
    let chart = am4core.create(node, am4plugins_wordCloud.WordCloud);
    let series = chart.series.push(new am4plugins_wordCloud.WordCloudSeries());
    series.text = 'one two three call is transferred';
```

Now, if I cause situation, where there is no words to show, for example by any of theese settings
```
    series.maxCount = 0;
    series.minWordLength = 40;
    series.minValue = 2;
```

I get error 
```
TypeError: Cannot read property 'label' of undefined
    at WordCloudSeries.processItem (WordCloudSeries.js?f820:334)
    at WordCloudSeries.validate (WordCloudSeries.js?f820:311)
    at eval (Container.js?be33:1671)
    at Module.each (Array.js?6e0f:74)
    at List.each (List.js?f2c0:723)
    at Container._systemUpdate (Container.js?be33:1662)
    at eval (System.js?72b5:241)
    at eval (Object.js?a0dd:107)
    at eachContinue (Object.js?a0dd:93)
    at Module.each (Object.js?a0dd:106)
```

Expect empty cloud without words and without error (see image)
Or need to know the result of filtering source text with options to decide - show I display cloud or not.

![image](https://user-images.githubusercontent.com/2210640/59805906-4dba1c80-92fb-11e9-8fdb-43767d1f4be2.png)

**Comment by martynasma** [comment]
Thanks for reporting this.

Simple repro example for dev team:
https://codepen.io/team/amcharts/pen/1664712d168fb09437db376d4a565734?editors=0010

**Comment by martynasma** [comment]
Fixed in [4.5.4](https://github.com/amcharts/amcharts4/releases/tag/4.5.4).

## [4.5.4] - 2019-07-08

### Added
- `allwaysShowTooltip` property added to `Sprite`. Allows tooltip to be shown without the need for hovering/selecting an element.
- `NumberFormatter` now supports `"!"` modifier which works in conjunction with `"a"` and `"b"` modifiers. If used, formatter will force application of a prefix/suffix for values that do not fit into any of the defined prefix lists. This is useful in cases you want to format all of the values using same prefix, e.g. "B" and want even small values to be formatted as such (`500,000` formatted as `"0.5B"`).

### Fixed
- `CategoryAxis` tooltip could show incorrect category if its `renderer.tooltipLocation` was set to `0`.
- Memory leak which happened when disposing a chart with `XYChartScrollbar` with series added to it fixed.
- Some series stacking issues fixed.
- Issue with `baseDuration.timeUnit = "yyyy"` and `StepLineSeries` fixed.
- In some cases `DateAxis` tooltips would show empty near edge of the plot area.
- `CategoryAxis.indexToPosition` adjusted to work better with no default `startLocation` and `endLocation` values.
- `WordCloud` could go into stackoverflow if `minWordLenght` was set to a number which resulted in no words to appear at all.
- `ForceDirectedSeries` were resulting in JS error if inited in a hidden container.
- If an axis range was made invisible by setting its `visible = false`, and then visibility was set back to `true` again, axis range was not shown.
- JSON: entries of type `OrderedListTemplate` or `SortedListTemplate` was not being processed correctly as templated lists.
- Legend items were still focusable/togglable via keyboard even if `clickable`/`focusable` on their respective containers were set to to `false`.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
